react-native/training/src/app/screens/TodoList/components/List/components/Item/layout.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import PropTypes from 'prop-types';
+
+import Checkbox from '../../../Checkbox';
+
+import styles from './styles';
+
+function Layout({ item, onToggleItem, onRemoveItem }) {
+  const itemStyle = item.completed ? [styles.item, styles.completed] : styles.item;
```

Comment:
You can replace this for: `[styles.item, item.completed ? styles.completed : ''];`